_pytest/helpconfig.py

```diff
@@ -3,13 +3,38 @@
 
 import py
 import pytest
+from _pytest.config import PrintHelp
 import os, sys
+from argparse import Action
+
+
+class HelpAction(Action):
+    def __init__(self,
```

Comment:
Could you add a short docstring here explaining this action? I guess basically what you wrote on the PR description would be great.